apps/vscode-extension/package.json

```diff
@@ -325,14 +328,14 @@
     "@o3r/dev-tools": "workspace:^",
     "@o3r/eslint-plugin": "workspace:^",
     "@o3r/schematics": "workspace:^",
+    "@types/eslint": "^8.42.0",
```

Comment:
is it needed?
It is not already provided in `eslint`?

Reply:
It doesn't seem to be provided